stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 7.4.0
+version: 7.5.0
```

Comment:
For each minor version bump, we should add to `UPGRADING.md` and update the link in `README.md`.

However, this could be considered a bug-fix, so I think bumping to `7.4.1` is probably acceptable.

Reply:
OK sounds good then for 7.4.1.